sqlmypy.py

```diff
@@ -141,14 +142,23 @@ def decl_info_hook(ctx):
 
         Base = declarative_base()
     """
+    cls_instances = []  # type: List[Instance]
+
+    if 'cls' in ctx.call.arg_names:
+        declarative_base_cls_arg = ctx.call.args[ctx.call.arg_names.index("cls")]
+        if isinstance(declarative_base_cls_arg, TupleExpr):
+            cls_instances = [Instance(item.node, []) for item in declarative_base_cls_arg.items]
+        else:
+            cls_instances = [Instance(declarative_base_cls_arg.node, [])]
```

Comment:
Here and above use `fill_typevars_with_any()`. If the base class turns to be generic, this manual construction of instances will cause crashes.